src/blockchain/examples/simple_node.rs

```diff
@@ -18,10 +18,255 @@ extern crate pretty_env_logger;
 extern crate pyrsia_blockchain_network;
 extern crate tokio;
 
-use pyrsia_blockchain_network::Example;
+use futures::StreamExt;
+use libp2p::{
+    core::upgrade,
+    floodsub::{self, Floodsub, FloodsubEvent},
+    identity,
+    mdns::{Mdns, MdnsEvent},
+    mplex,
+    noise,
+    swarm::{NetworkBehaviourEventProcess, SwarmBuilder, SwarmEvent},
+    // `TokioTcpConfig` is available through the `tcp-tokio` feature.
+    tcp::TokioTcpConfig,
+    Multiaddr,
+    NetworkBehaviour,
+    PeerId,
+    Transport,
+};
+use rand::Rng;
+use std::error::Error;
+use tokio::io::{self, AsyncBufReadExt};
```

Comment:
Following christians idea we can skip this name

```suggestion
use tokio::io::{self, AsyncBufReadExt};
use pyrsia_blockchain_network::*;
```